src/order.cpp

```diff
@@ -1242,6 +1242,9 @@ static void orderPlayFireSupportAudio(BASE_OBJECT *psObj)
 	}
 }
 
+inline bool isRetreating(const DROID *psDroid) {
+	return(psDroid->order.type == DORDER_RTR || psDroid->order.type == DORDER_RTB || psDroid->order.type == DORDER_RTR_SPECIFIED);
```

Comment:
By doing so, you make ALL the droids to ignore orders, not only those that have a group assigned. You should check for their group

Reply:
Agreed, will fix.